README.md

```diff
@@ -175,8 +208,15 @@ The following options are supported:
 |glad             |flag  |all                      |Filter input feature by GLAD tile extent, requires boolean `glad` field in input feature class                                   |
 |change_only      |flag  |all except `treecover`   |Process change only                                                                                                              |
 |sensitivity_type |string|`carbon_sensitivity`     |Select carbon sensitivity model                                                                                                  |
-|fire_alert_type  |string|`firealerts`             |Select Fire alert type                                                                                                           |
-|fire_alert_source|string|`firealerts`             |URI of fire alert TSV file                                                                                                       |
+|fire_alert_type  |string|`firealerts` |Select Fire alert type                                                                                                                       |
+|fire_alert_source|string|`firealerts` |URI of fire alert TSV file                                                                                                                   |
+|overwrite        |flag  |all                      |Overwrite output location if already existing                                                                                    |
+|split_features   |flag  |all                      |Split input features along 1x1 degree grid                                                                                       |
+|no_output_path_suffix|flag |all                   |Do not autogenerate output path suffix at runtime                                                                                |
+|intermediate_list_source|flag |`forest_change_diagnostic` analysis |URI of intermediate list results in TSV format                                                                  |
+|contextual_feature_type|flag |`gfwpro_dashboard` analysis |URI of intermediate list results in TSV format                                                                           |
```

Comment:
copy and error here, this should be the type of feature for the contextual data

Reply:
Fixed, thanks for catching this!